Fix:
error: invalid operands to binary expression ('__bind<int &, sockaddr *&, unsigned int &>' and 'int')
https://lists.gnu.org/archive/html/bug-apl/2021-04/msg00004.html
http://svn.savannah.gnu.org/viewvc/apl/trunk/src/main.cc?r1=1467&r2=1466&pathrev=1467
--- src/main.cc.orig	2019-06-23 07:39:20.000000000 -0500
+++ src/main.cc	2022-01-12 20:32:24.000000000 -0600
@@ -28,6 +28,7 @@
 #include <sys/ioctl.h>
 #include <sys/resource.h>
 #include <sys/stat.h>
+#include <sys/socket.h>
 #include <sys/types.h>
 
 #include "buildtag.hh"
